lib/super_digit.rb

```diff
@@ -1,15 +1,18 @@
 # Superdigit
 
-# Time Complexity - ?
-# Space Complexity - ?
+# Time Complexity - O(logn)
+# Space Complexity - O(logn)
 def super_digit(n)
```

Comment:
Since you're summing an doing the recursion inside the loop, this is O( (log n)^2)